package.json

```diff
@@ -53,11 +53,11 @@
     "eslint": "^7.6.0",
     "eslint-config-nebenan": "^3.0.0",
     "mocha": "^8.2.1",
-    "react": "^17.0.1",
+    "react": "^18.2.0",
```

Comment:
are the react devDependencies needed at all? dont see any tests or other dev stuff using them explicitly....

Reply:
Hmm you're right, looking at the history it was used back in the days but I also cannot find any usage...